app/views/works/show.html.erb

```diff
@@ -248,21 +248,11 @@
       <% end %>
     </dl>
 
-    <% if @work.approved? %>
-      <% if @work.post_curation_s3_resources.present? %>
-        <section class="uploads">
-          <h2 class="h2"><%= t('works.uploads.post_curation.heading') %></h2>
-          <%= render(partial: 'works/uploads/post_curation_s3_resources') %>
-        </section>
-      <% end %>
-    <% else %>
-      <% if @work.pre_curation_uploads_fast.present? %>
-        <section class="uploads">
-          <h2 class="h2"><%= t('works.uploads.pre_curation.heading') %></h2>
-          <%= render(partial: 'works/uploads/pre_curation_s3_resources') %>
-        </section>
-      <% end %>
-    <% end %>
+    <section class="uploads">
+      <h2 class="h2"><%= @work.approved? ? "Curated Files" : "Uncurated Files" %></h2>
+      <%= render(partial: 'works/s3_resources') %>
+    </section>
+
```

Comment:
`works/s3_resources` partial is the merged version of `_post_curation_s3_resources` and `_pre_curation_s3_resources`